packages/xsnap/src/build.js

```diff
@@ -23,6 +23,31 @@ function exec(command, cwd, args = []) {
 }
 
 (async () => {
+  // Detect whether we're under Git.  We aren't when building Docker images.
+  let underGit;
+  try {
+    await exec('git', '.', ['submodule']);
+    underGit = true;
+  } catch (e) {
+    underGit = false;
+  }
+
+  // Do the moral equivalent of submodule when not under Git.
+  // TODO: Make this less fragile.
```

Comment:
less fragile how?

maybe you mean: TODO: refactor overlap with git submodules file?

Reply:
> maybe you mean: TODO: refactor overlap with git submodules file?

Yup!  Noted.